canonical-web-and-design/ubuntu.com
# Copy update on financial services

## Done

- Copy update on `/financial-services` - added link to hero and updated 'Software strategy acceleration' section

## QA

- View page at: https://ubuntu-com-9214.demos.haus/financial-services
- Run through the following [QA steps](https://canonical-web-and-design.github.io/practices/workflow/qa-steps.html)
- Check page against [copy doc](https://docs.google.com/document/d/1SEd2nV0GplBTG4lZ9RuhjNXrphU22nGP4Fpgd5g_9BQ/edit?ts=601c07bc#)


## Issue / Card

Fixes [#3795](https://github.com/canonical-web-and-design/web-squad/issues/3795)

## Screenshots

![image](https://user-images.githubusercontent.com/58959073/107214962-95c31100-6a02-11eb-955f-3b28c63eddc7.png)
![image](https://user-images.githubusercontent.com/58959073/107214970-99569800-6a02-11eb-871b-4454f96f1676.png)
